### PR TITLE
Fix/358 add missing expanded attributes

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**

### CODE REVIEW

File: apps/files/js/fileactions.js [hunk 1]
```diff
@@ -370,6 +370,7 @@
 			var menu;
 			var $trigger = context.$file.closest('tr').find('.fileactions .action-menu');
 			$trigger.addClass('open');
+			$trigger.attr('aria-expanded', "true");
```
Comment (PVince81) [inline apps/files/js/fileactions.js:373]: mixing single quotes with double quotes ? please change to single quotes then

File: apps/theming/js/3rdparty/jscolor/jscolor.js [hunk 2]
```diff
@@ -1354,13 +1354,15 @@ var jsc = {
 
 
 		function detachPicker () {
+			THIS.targetElement.setAttribute('aria-expanded', 'false');
```
Comment (PVince81) [inline apps/theming/js/3rdparty/jscolor/jscolor.js:1357]: should `THIS` be lower case ? I didn't scroll up far enough to find its definition

also: should we probably submit an upstream patch for this

File: webpack.common.js [hunk 3]
```diff
@@ -79,10 +79,10 @@ module.exports = {
 			{
 				test: /\.vue$/,
 				loader: 'vue-loader',
-				exclude: BabelLoaderExcludeNodeModulesExcept([
-					'vue-material-design-icons',
-					'emoji-mart-vue-fast',
-				]),
+				// exclude: BabelLoaderExcludeNodeModulesExcept([
```
Comment (PVince81) [inline webpack.common.js:82]: workaround checked it by mistake, please revert :smile:

File: apps/systemtags/js/admin.js [hunk 4]
```diff
@@ -125,6 +132,7 @@
 		 * @param {number} tagId
 		 */
 		_prepareForm: function (tagId) {
+			console.log($('#systemtags').children('.select2-container'))
```
Comment (juliushaertl) [inline apps/systemtags/js/admin.js:135]:
```suggestion
```

Debugging leftover?

Comment (JuliaKirschenheuter) [reply apps/systemtags/js/admin.js:135]: yes, sorry

File: apps/theming/js/3rdparty/jscolor/jscolor.js (hunk 2 above)
Comment (JuliaKirschenheuter) [reply apps/theming/js/3rdparty/jscolor/jscolor.js:1357]: i've corrected it. but yes, i don't know why, but in this file ```this``` will be used as ```THIS```

Comment (JuliaKirschenheuter) [reply apps/theming/js/3rdparty/jscolor/jscolor.js:1357]: ```this``` should be used like ```THIS```. otherwise it not works

**Review by Pytal — APPROVED** [review]

File: apps/files/templates/appnavigation.php [hunk 5]
```diff
@@ -91,6 +91,8 @@ class="nav-icon-<?php p(isset($item['icon']) && $item['icon'] !== '' ? $item['ic
 	if (isset($item['sublist'])) {
 		?>
 			<button class="collapse app-navigation-noclose"
+				<?php if (isset($item['defaultExpandedState']) && $item['defaultExpandedState']) { ?> aria-expanded='true'<?php }
+				else { ?> aria-expanded='false'<?php } ?>
```
Comment (kesselb) [inline apps/files/templates/appnavigation.php:95]:
```suggestion
				aria-expanded="<?= !empty($item['defaultExpandedState']) ? 'true' : 'false' ?>"
```

File: apps/theming/js/3rdparty/jscolor/jscolor.js (hunk 2 above)
Comment (skjnldsv) [reply apps/theming/js/3rdparty/jscolor/jscolor.js:1357]: What the hell :sweat_smile: 
Let get this merge, it was clearly here before! :see_no_evil:

**Review by skjnldsv — APPROVED** [review]

**Review by ChristophWurst — APPROVED** [review]
:+1: :elephant:

File: apps/files/templates/appnavigation.php (hunk 5 above)
Comment (skjnldsv) [reply apps/files/templates/appnavigation.php:95]: @JuliaKirschenheuter

**Pull request merged**